strings/br.py

```diff
@@ -0,0 +1,433 @@
+# Strings / localization file for greed
+# Can be edited, but DON'T REMOVE THE REPLACEMENT FIELDS (words surrounded by {curly braces})
+
+# Part of the translation by https://github.com/DarrenWestwood
+
+# Currency symbol
+currency_symbol = "R$"
+
+# Positioning of the currency symbol
+currency_format_string = "{symbol} {value}"
+
+# Quantity of a product in stock
+in_stock_format_string = "{quantity} Disponível"
```

Comment:
```suggestion
in_stock_format_string = "{quantity} disponível"
```

This should probably be lowercase.